Core/src/IfcGeometryConverter/CurveConverter.h

```diff
@@ -855,8 +855,79 @@ namespace OpenInfraPlatform {
 					GeomUtils::appendPointsToCurve(basisCurvePoints, targetVec);
 				}
 
-				// IfcClothoid SUBTYPE of IfcCurve (exists starting IFC4X3_RC3)
+				// IfcSpline SUBTYPE of IfcCurve (exists starting IFC4X3_RC3)
 #if defined(OIP_MODULE_EARLYBINDING_IFC4X3_RC3)
+				/**********************************************************************************************/
+				/*! \brief Converts an \c IfcSpiral to a tesselated curve.
+				* \param[in] spiral					A pointer to data from \c Spiral.
+				* \param[out] targetVec				The tessellated line.
+				* \param[out] segmentStartPoints	The starting points of separate segments.
+				* \param[in] trim1Vec				The trimming of the curve as saved in IFC model - trim at start of curve.
+				* \param[in] trim2Vec				The trimming of the curve as saved in IFC model - trim at end of curve.
+				* \param[in] senseAgreement			Does the resulting geometry have the same sense agreement as the \c IfcSpiral.
+				*/
+				void convertIfcSpiral(
+					const EXPRESSReference<typename IfcEntityTypesT::IfcSpiral>& spiral,
+					std::vector<carve::geom::vector<3>>& targetVec,
+					std::vector<carve::geom::vector<3>>& segmentStartPoints,
+					const std::vector<std::shared_ptr<typename IfcEntityTypesT::IfcTrimmingSelect>>& trim1Vec,
+					const std::vector<std::shared_ptr<typename IfcEntityTypesT::IfcTrimmingSelect>>& trim2Vec,
+					const bool senseAgreement,
+					const typename IfcEntityTypesT::IfcTrimmingPreference & trimmingPreference
+				) const throw(...)
+				{
+					// **************************************************************************************************************************
+					//	 https://
+					//	ENTITY IfcSpiral
+					//		ABSTRACT SUPERTYPE OF(ONEOF
+					//			(IfcBlossCurve,
+					//          IfcClothoid
+					//          IfcCosine
+					//          IfcHelmertCurve
+					//			IfcSine))
+					//		SUBTYPE OF(IfcCurve);
+					//		Position: OPTIONAL IfcAxis2Placement;
+					//	END_ENTITY;
+					// **************************************************************************************************************************
+					// IfcSpiral SUBTYPE OF IfcBlossCurve

```

Comment:
```suggestion
					// IfcBlossCurve SUBTYPE OF IfcSpiral  
```

It is the other way around.

Other similar comments in this function are wrong as well.

Reply:
Resolved  875f326